components/IssueCardColumn.vue

```diff
@@ -0,0 +1,15 @@
+<script lang="ts" setup>
+defineProps<{ title: string }>()
+</script>
+
+<template>
+  <div
+    class="min-w-[350px] max-w-[350px] flex-1 rounded-[4px] border border-rad-border-hint bg-rad-background-dip"
```

Comment:
`min-w-something` and `max-w-something` can be replaced with `w-something`

Also, id ti possible that we use one of those instead of rolling our own?

w-80	width: 20rem; /* 320px */
w-96	width: 24rem; /* 384px */

Reply:
1. Unfortunately since we're in a flex layout it can't, [width isn't a hard constraint in flex (PS one of the best people I follow for CSS)](https://www.joshwcomeau.com/css/interactive-guide-to-flexbox/#hypothetical-size-5).

2. As for the second point, `min-w-` doesn't have any defaults that would be applicable here.

FYI: I referenced GitHub's columns for the 350px.

Thoughts?